editor/components/rich-text/index.js

```diff
@@ -405,10 +405,14 @@ export class RichText extends Component {
 	/**
 	 * Handles any case where the content of the tinyMCE instance has changed.
 	 */
-
 	onChange() {
+		const { name, onChange, setAttribute } = this.props;
```

Comment:
I feel our RichText component is getting bigger with more props. Maybe we can divide RichText into a more generic low-level version that does not use context. And a higher level version that uses context. When passing name the higher level version would just pass value + onChange to the low-level version. I feel this might simplify things and make it easier to add more high-level abstractions in the future.

Reply:
Yes, that's a good observation. It's definitely something we should explore soon to make sure `RichText` doesn't reach 1k lines :)